packages/app-form-builder/tsconfig.json

```diff
@@ -7,6 +7,9 @@
     {
       "path": "../app-admin"
     },
+    {
```

Comment:
Changes in this file are not necessary, because you're already within that package. There's no need to add mapping for "self".

Reply:
Okay. I will revert that change.